test/e2e/apps/daemon_set.go

```diff
@@ -421,7 +421,7 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 	framework.ConformanceIt("should rollback without unnecessary restarts", func() {
 		schedulableNodes, err := e2enode.GetReadySchedulableNodes(c)
 		framework.ExpectNoError(err)
-		gomega.Expect(len(schedulableNodes.Items)).To(gomega.BeNumerically(">", 1), "Conformance test suite needs a cluster with at least 2 nodes.")
+		//gomega.Expect(len(schedulableNodes.Items)).To(gomega.BeNumerically(">", 1), "Conformance test suite needs a cluster with at least 2 nodes.")
```

Comment:
@ravisantoshgudimetla it would be best to sync with upstream, especially conformance testing group. Do we want to have limitation on skips for clusters with one node where this test isn't of much value.

Reply:
Ya, it is on my TODO for today. I opened this PR to see if it is working this way,